packages/snaps-rpc-methods/src/restricted/notify.ts

```diff
@@ -133,11 +184,24 @@ export function getImplementation({
       context: { origin },
     } = args;
 
-    const validatedParams = getValidatedParams(params);
-
     await maybeUpdatePhishingList();
 
-    validateTextLinks(validatedParams.message, isOnPhishingList, getSnap);
+    const validatedParams = getValidatedParams(
+      params,
+      isOnPhishingList,
+      getSnap,
+    );
+
+    let id;
```

Comment:
This `id` doesn't seem to be used elsewhere. Can it just be a const variable declared inside the if block?